athena-postgresql/src/main/java/com/amazonaws/athena/connectors/postgresql/PostGreSqlMetadataHandler.java

```diff
@@ -143,6 +147,14 @@ public GetDataSourceCapabilitiesResponse doGetDataSourceCapabilities(BlockAlloca
         ));
 
         jdbcQueryPassthrough.addQueryPassthroughCapabilityIfEnabled(capabilities, configOptions);
+
+        //Provide a hint to the engine that postgresql is using default collate settings
+        //Which doesn't match Athena's Engine Collation; this disabling Predicate pushdown
+        boolean usingDefaultCollate = Boolean.valueOf(this.configOptions.getOrDefault(USING_DEFAULT_COLLATE, "false"));
```

Comment:
Is the default boolean for "using_default_collate" supposed to be false? The naming seems confusing here. It looks like capabilities is using NON_DEFAULT_COLLATE setting when using_default_collate is true. Is this intended?

Reply:
Using default collate is supposed to be false by default so we don't effect existing customers; and customer who chose to use PostgreSQL default collation; required to set this to true in order to disable dynamic filtering.

Reply:
Can we make the name more explicit for it seems like using_default_collate = true is enabling a NON_DEFAULT_COLLATE. Maybe using_postgresql_default_collate? or can change NON_DEFAULT_COLLATE naming

Reply:
changed the name to be clearer;